src/main/ApplicationUtils.cpp

```diff
@@ -50,6 +50,12 @@ runWithConfig(Config cfg)
     {
         app = Application::create(clock, cfg, false);
 
+        if (cfg.OP_APPLY_SLEEP_TIME_FOR_TESTING > 0)
```

Comment:
why do we need this?

Reply:
we're spinning up a new network and using in-memory db

Reply:
That doesn't seem related to `OP_APPLY_SLEEP_TIME_FOR_TESTING`:
I think it's probably cleaner to adjust the config and set `FORCE_SCP` to `true` if the node is configured to be a validator and we're in sqlite in memory mode. The adjustment should be done at line 45 (after the other place where we do this kind of adjustment)